src/tabs/tabTemplate.jsx

```diff
@@ -8,16 +8,14 @@ class TabTemplate extends React.Component {
 
   render() {
     const styles = {
-      height: 0,
-      overflow: 'hidden',
       width: '100%',
       position: 'relative',
       textAlign: 'initial',
     };
 
-    if (this.props.selected) {
-      delete styles.height;
-      delete styles.overflow;
+    if (!this.props.selected) {
```

Comment:
Haha, believe it or not, these were the only two uses of `delete` in our entire code base. We should create an umbrella issue for this and close it and marvel at its glory. :laughing: